src/contentscript/overlay/root/components/OverlayToolbar/index.tsx

```diff
@@ -87,15 +96,50 @@ export const OverlayToolbar = (p: OverlayToolbarProps): ReactElement => {
   const [isVisibleAnimation, setVisibleAnimation] = useState(false)
   const [iconAnimateWidget, setIconAnimateWidget] = useState('')
   const [isPinnedAnimateWidget, setPinnedAnimateWidget] = useState(false)
+  const [isPinnedNotification, setPinnedNotification] = useState(false)
+  const [event, setEvent] = useState<Notify[]>([])
+  const [payload, setPayload] = useState(null)
   const btnRef = useRef<HTMLDivElement>()
+  const notificationRef = useRef<HTMLDivElement>()
+
   useEffect(() => {
     const init = async () => {
       await _refreshData()
+      // await  handleUpdateNotifications()
     }
 
     init()
+
     return () => {}
-  }, [newWidgets, widgets, nodeOverlayToolbar, isClick])
+  }, [newWidgets, widgets, isClick])
+  const handleUpdateNotifications = async () => {
+    const notifications = await getNotifications()
+    setEvent(notifications && notifications.filter((x) => x.status === 1))
+  }
+  useEffect(() => {
+    const updatePinnedNotifications = async (payload) => {
+      const notifications = await getNotifications()
+      payload && setPinnedNotification(true)
+      setPayload(payload)
+
+      setTimeout(() => {
+        setPinnedNotification(false)
+      }, 8000)
+      return setEvent(notifications)
+    }
+
+    EventBus.on('notifications_updated', handleUpdateNotifications)
+    EventBus.on('show_notification', (payload) => {
```

Comment:
Is notifications_updated not enough?

Reply:
not enough
Animate is used when notification show